core/build.gradle

```diff
@@ -59,6 +27,51 @@ task sourcesJar(type: Jar, dependsOn: classes) {
     from sourceSets.main.allSource
 }
 
+publishing {
+    publications {
+        corePublication(MavenPublication) {
+            from components.java
+            artifact sourcesJar
+            pom {
+                groupId = "org.p5android"
+                artifactId = "processing-core"
+                version = "${modeVersion}"
+                packaging = "jar"
+                // description = "Processing Android Core"
+                // url = "http://www.example.com/project"
+                licenses {
+                    license {
+                        name = "GNU Lesser General Public License, version 2.1"
+                        url = "https://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt"
+                        distribution = "repo"
+                    }
+                }
+            }
+            pom.withXml {   // Only one dependency is added under dependancies node
+                asNode().remove(asNode().get("dependencies")) // removing dependencies node
+                // inserting the dependencies node
+                def dependenciesNode = asNode().appendNode('dependencies')
+                // start adding dependency nodes inside dependencies node
+                def dependencyNode = dependenciesNode.appendNode('dependency')
+                dependencyNode.appendNode('groupId', 'androidx.legacy')
+                dependencyNode.appendNode('artifactId', 'legacy-support-v4')
+                dependencyNode.appendNode('version', "${v4legacyVersion}")
+                dependencyNode.appendNode('scope', 'implementation')
+
+                def dependencyNode2 = dependenciesNode.appendNode('dependency')
```

Comment:
I think you should follow proper naming of variables !

ex - instead of dependencyNode you can use something like androidLegacyDependancyNode
and instead of dependencyNode2 you can use wearableDependencyNode

Proper naming of variables is very important in any OpenSource project !

Reply:
Corrected variable names in Core build.gradle file, Thanks for providing a clear example!